src/components/ProfileSettings/ProfileSettings.js

```diff
@@ -0,0 +1,218 @@
+import React, { useState } from 'react';
+
+const ProfileSettings = () => {
+  const [location, setLocation] = useState('')
+  const [pronoun, setPronoun] = useState('')
+  const [bio, setBio] = useState('')
+  const [selectedInterests, setSelectedInterests] = useState([])
+  const [interests, setInterests] = useState({
+    sports: false,
+    nature: false,
+    music: false,
+    night_life: false,
+    art: false,
+    cinema: false,
+    food: false,
+    video_game: false,
+    traveling: false,
+    networking: false
+  })
+  
+  const updateLocation = (e) => {
+    setLocation(e.target.value)
+  }
+
+  const updatePronoun = (e) => {
+    setPronoun(e.target.value)
+  }
+
+  const updateBio = (e) => {
+    setBio(e.target.value)
+  }
+
+  const updateInterests = (e) => {
+    if (e.target.checked) {
+      setInterests({...interests, [e.target.value]: true})
+      setSelectedInterests([...selectedInterests, e.target.value])
+    } else {
+      setInterests({ ...interests, [e.target.value]: false })
+      const removedIndex = selectedInterests.indexOf(e.target.value)
+      setSelectedInterests([
+        ...selectedInterests.slice(0, removedIndex),
+        ...selectedInterests.slice(removedIndex + 1)
+      ])
+    }
+  }
+
+  const submitSettings = (e) => {
+    e.preventDefault()
+  }
+
+  return(
+    <div>
+      <h1>Edit Your Profile</h1>
+      <form onSubmit={submitSettings}>
+        <label htmlFor='location'>Change location: </label>
+        <input
+          type='text'
+          placeholder='Denver, CO'
+          id='location'
+          name='location'
+          value={location}
+          onChange={(e) => updateLocation(e)}
+        />
+
+        <p>Would you like to update your pronouns?</p>
```

Comment:
For pronouns I think we should maybe do a select box like we did for interests to allow users to select multiple pronouns if they want, but not a huge priority as long as it's functional